st_cc.py

```diff
@@ -164,6 +164,35 @@ def get_opt(view):
     project_settings = view.settings()
     include_opts = settings.get("include_options", []) + project_settings.get("cc_include_options", [])
 
+    # seach for .clang_complete file and add lines to options
+    target_file = ".clang_complete"
+    directory = os.path.dirname(view.file_name())
```

Comment:
Won't this not work with nested structures? The `.clang_complete` file is typically in the root of the project but what if some of my code is in `lib/` or `src/`. Maybe it would be better to use `os.path.dirname(sublime.active_window().folders()[0])` which will be the main folder of the current window.